datastore/src/main/java/io/spine/server/storage/datastore/DatastoreWrapper.java

```diff
@@ -274,11 +273,13 @@ public void createOrUpdate(Collection<Entity> entities) {
      * @see DatastoreReader#run(Query)
      */
     public <R> DsQueryIterator<R> read(StructuredQuery<R> query) {
-        Namespace namespace = currentNamespace();
+        Namespace namespace = namespaceSupplier.get();
         StructuredQuery<R> queryWithNamespace =
                 query.toBuilder()
                      .setNamespace(namespace.getValue())
                      .build();
+        _trace().log("Reading entities of `%s` kind with `%s` filter in `%s` namespace.",
```

Comment:
```suggestion
        _trace().log("Reading entities of kind: `%s` in namespace: `%s` .",
```

Reply:
Please also adjust the arguments line.

Reply:
Please also apply to the other logs in this file.

Reply:
As discussed vocally, we'd better avoid additional special characters in logs as it generally makes logs filtering harder in e.g. Stackdriver